docs/input4MIPs_files_CMIP6Plus.html

```diff
@@ -43973,6 +43973,264 @@
       <td>20240718</td>
       <td>None</td>
     </tr>
+    <tr>
```

Comment:
If files don't exist, I see little point in including blank entries

Reply:
Can make that tweak. That means that there will be no files with a status of 'registered', I assume that's something you're happy with?

Reply:
Yeah I think so, as until files appear we're not sure what we're going to get, as some timeseries might be separated into two or more files etc - there will be no sha, and just a whole bunch of null entries

Reply:
Ok cool, see https://github.com/PCMDI/input4MIPs_CVs/pull/79